### ISSUE TITLE
What data is the heatmap representing exactly?

### ISSUE THREAD

**Issue by flogy** [spring-labs/skill-reader#6]
The skills that can be retrieved using the LinkedIn interface [are just Strings](https://developer.linkedin.com/docs/fields/skills). So the question I ask myself is, what should the heatmap actually show?

**Comment by openwms** [comment]
Yes that's the point. Probably we need to deal with the LinkedIn API directly and not with spring-social-linkedin. Have to read the API docs to find out if there's a possibility to get the number of skills
